### PR TITLE
style: redesign `MyList`

### PR DESCRIPTION
**Pull request by tokitou-san**

### CODE REVIEW

**Comment by tokitou-san** [comment]
Whatya think about replacing tippy with this custom like popup?
Reason: Tippy is shown around image... it looks good if we can show tippy on center like this.

Screenshot from 2023-08-11 18-09-53

**Comment by baseplate-admin** [comment]
>  Reason: Tippy is shown around image... it looks good if we can show tippy on center like this. 

We can

**Comment by baseplate-admin** [comment]
Untitled


Follow this approach

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -519,72 +520,37 @@
...
+                                </dubs>
+                            </div>
+                        </overlay>
+                    </anime-cover>
+
+                    <anime-details class="flex flex-col md:gap-[0.5vw]">
+                        <anime_name class="line-clamp-1 text-base font-semibold md:text-[1.1vw] leading-none">{anime.name}</anime_name>
+                        <anime_info class="flex items-center gap-2 text-sm text-surface-50 md:gap-[0.5vw] md:text-[0.9vw] leading-none">
+                            <span>Watching</span>
+                            <Circle class="w-1 md:w-[0.25vw]" />
```
Comment (baseplate-admin) [inline src/routes/+page.svelte:550]: Why use `Circle`?

Comment (tokitou-san) [reply src/routes/+page.svelte:550]: Why not?
`after:content` this is hard to maintain

Comment (baseplate-admin) [reply src/routes/+page.svelte:550]: I still like `after:content`

Comment (tokitou-san) [reply src/routes/+page.svelte:550]: > I still like after:content

I cant go with `after::`
wanna use `RoundSquare` instead?

Comment (baseplate-admin) [reply src/routes/+page.svelte:550]: No use `<Tattoo />`

**Pull request merged**